python_hw_1/calc.py

```diff
@@ -0,0 +1,50 @@
+def addition(number1, number2):  # addition function
+    return f"The result of addition two numbers is: {number1 + number2}"
+
+
+def subtraction(number1, number2):  # subtraction function
+    return f"The result of subtraction two numbers is: {number1 + number2}"
+
+
+def multiplication(number1, number2):  # multiplication function
+    return f"The result of multiplication two numbers is: {number1 * number2}"
+
+
+def division(number1, number2):  # division function
+    try:
+        result = number1 / number2
+        return f"The result of division two numbers is: {result}"
+    except ZeroDivisionError:
+        return "Second number can't be zero!"
+
+
+number_1 = int(input("Please enter the first number: "))
+print()
+
+number_2 = int(input("Please enter the second number: "))
+print()

```

Comment:
you can add \n in your str to add new line
Dont use print() as new liner)

Reply:
Changes was made